lib/Importer/ONNX.cpp

```diff
@@ -364,7 +375,10 @@ void ONNXModelLoader::loadOperator(const onnx::NodeProto &op) {
     Node *B = getOrCreateVariableByName(op.input(1));
     Node *C = getOrCreateVariableByName(op.input(2));
 
-    bool broadcastC = dict.count("broadcast") && loadInt(dict["broadcast"]);
+    bool broadcastC =
+        (opset_version == 1 || opset_version == 6)
```

Comment:
what about opset_version 2 or 3, etc? I think it's more readable this way:
```
bool broadcastC = true;
if (opset_version <= 6)
  broadcastC = dict.count("broadcast") && loadInt(dict["broadcast"]);
```

Reply:
Well, opset_version is not coming from Gemm's description. It's common for all operators. I think that it can be 0 or 3 (because of some other op's changes), and we can totally see any number here.